src/peft/tuners/lora/model.py

```diff
@@ -540,7 +540,9 @@ def add_weighted_adapter(
             adapters (`list`):
                 List of adapter names to be merged.
             weights (`list`):
-                List of weights for each adapter.
+                List of weights for each adapter. Weights can be positive or negative, allowing for both addition
+                and subtraction of adapter effects. Negative weights enable task arithmetic and adapter negation as
+                described in the LoRA Hub and Task Arithmetic papers.
```

Comment:
I would cut this, e.g. for `_svd_generalized_task_arithmetic_weighted_adapter` in [SVD-based merging](https://github.com/huggingface/peft/blob/ce5c2044f1430aa0bb3d2bccda73e242efde9e72/src/peft/tuners/lora/model.py#L775), this was already available and I don't think the hint to LoraHub or Task Arithemetic neccessary here

```suggestion
                List of weights for each adapter.
```